contribution_guidelines.md

```diff
@@ -0,0 +1,253 @@
+# Contributing to the Neuroimaging Cookbook
```

Comment:
This is great, but honestly I would remove what we can't say/describe right now, personalize what already fit us, and then progressive add piece by piece. I worry of the over-engineering and overwhelming effect that has on newbies